TODO.md

```diff
@@ -1,10 +1,5 @@
 # For random little notes of things I can't forget
 
-*. Setup github actions -> slack  
```

Comment:
shouldn't we just remove the TODO.md file entirely?

Reply:
Sure. This was actually a test PR when I was having issues with the integration tests. :)